web/themes/joinup/templates/form/form-element--checkbox.html.twig

```diff
@@ -82,13 +74,13 @@
       {{ description.content }}
     </div>
   {% endif %}
-  {{ children }}
+  <label class="mdl-checkbox mdl-js-checkbox mdl-js-ripple-effect">
```

Comment:
Hmm, I do not understand. It is not password template but form-element template.